kubectx

```diff
@@ -103,10 +106,12 @@ switch_context() {
 }
 
 choose_context_interactive() {
+  [[ -f ${CONTEXT_LIST_CACHE} ]] && HEIGHT=$(($(eval cat ${CONTEXT_LIST_CACHE} | wc -l) + 2)) || HEIGHT="100%"
```

Comment:
1. you don't need `eval` in `$(eval cat ...)`

2. please unfold `[[ -f ]] && ` as `if [..`

3. I think you should create a method named `context_list_count` and call it from here instead of embedding the file logic here.

4. I think there's a bug here. first time I call it (with no `.cache` dir), I get 100%, but the next time it works. It appears like the HEIGHT doesn't reflect the current count, but the previous run. This is a problem when I just added a new 5th context, but next time I run kubectx, height will be =4 (and so it won't show one of the contexts).

Reply:
the solution to #4 might be to actually modify SELF_CMD to use command `cat $cache_file`.

because if we just computed the cache, it's probably cool.